test/functional/wallet_basic.py

```diff
@@ -404,8 +404,6 @@ def run_test(self):
             '-reindex',
             '-zapwallettxes=1',
             '-zapwallettxes=2',
-            # disabled until issue is fixed: https://github.com/bitcoin/bitcoin/issues/7463
-            # '-salvagewallet',
```

Comment:
Would be nice to keep this test or at least document that salvagewallet will throw away your keys: #7379

Reply:
> @MarcoFalke could you describe the test or comment you would add? It sounds there is a good suggestion here, but I don't understand it's asking for

The new functionality has zero tests. I was hoping we could call `wallet-tool salvage` instead of `bitcoind -salvagewallet` here. Just because the test is commented out, doesn't mean it should be removed.

If the test still fails occasionally with the wallet-tool, then it can be commented out again.

Reply:
The behavior of salvage shouldn't be any different, so it will probably still fail as the previous `-salvagewallet` did.

Reply:
I've added a placeholder test with a todo that mentions the issue in `tool_wallet.py`.